hdfs3/tests/test_example.py

```diff
@@ -0,0 +1,23 @@
+from hdfs3 import HDFileSystem
+
+def test_example():
+    import time
+    import os
+    os.environ["LIBHDFS3_CONF"] = "/etc/hadoop/conf/hdfs-site.xml"
```

Comment:
I don't think its needed for this test but i its needed for something, short circuit reads? I moved it to the `Dockerfile`